tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -189,28 +189,6 @@ describe('Header', () => {
     expect(welcomeTab.hasClass('usa-current')).toEqual(false)
   })
 
-  it('should log out user when sign out button is clicked', () => {
```

Comment:
All of the removed tests below are now invalid since this link has been entirely removed.